nuxeo-core/nuxeo-core-el/src/main/java/org/nuxeo/ecm/platform/el/DocumentModelResolver.java

```diff
@@ -249,6 +276,21 @@ public void setValue(ELContext context, Object base, Object property, Object val
                 }
             }
             context.setPropertyResolved(true);
+        } else if (base instanceof Blob) {
+            // handle the Blob use case too here instead of a dedicated EL resolver, to avoid multiplying resolvers in
+            // the chain
+            try {
+                super.setValue(context, base, property, value);
+            } catch (PropertyNotFoundException e) {
+                Blob blob = (Blob) base;
+                if ("name".equals(property)) {
+                    value = blob.getFilename();
+                    context.setPropertyResolved(true);
+                } else if ("mime-type".equals(property)) {
+                    value = blob.getMimeType();
+                    context.setPropertyResolved(true);
```

Comment:
This looks copy/pasted from `getValue`, but shouldn't it set the value instead? If so, a unit test is also missing.

Reply:
Reworked this part, added more tests and re-enabled another, relaunched tests: https://qa.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-atchertchian-master/151/